gis/text_to_centreline_geometry_scripts/README.md

```diff
@@ -0,0 +1,189 @@
+
+The function outputs a confidence level (`con`), which is defined after the `centreline_segments` variable. This value represents how close the names of the input values are to the intersections they were matched to. Specifically, this value is the sum of the levenshtien distance between the `highway2` value and the street name it matched to and the `btwn1`/`btwn2` value and the street name it matched to. It is the sum of this value for both intersections.
+
+# Special Cases
+
+There are some records that start and/or end at locations that are a certain amount of metres away from an intersection. We create a method to assign a geometry to the locations of these bylaws. These segments do not start at one intersection and end at another, they often start/end in the middle of centreline segments.
+
+### Explore the data/cases
+
+**Case 1**
+
+btwn2 is formatted like: "a point (insert number here) metres (direction - north/south/east/west)"
+
+The point is located on the street in the highway column a certain number of metres away from the intersection identified in btwn1.
+
+These records can be filtered with the WHERE clause: `btwn2 LIKE '%point%'`
+
+There are 23 records with this case.
+
+**Case 2**
+
+btwn1 or btwn2 formatted like: "(number) metres (direction) of (insert street name that is not btwn1 or highway)"
+
+The point is located a certain number if metres away from the specified intersection. The intersection is an intersection between the street in the highway column and the (insert street name that is not btwn1 or highway).
+
+These records can be filtered with the WHERE clause: `btwn LIKE '%metres%of%`
+
+Example:
+street = "Watson Avenue"
+btwn = "Between St Marks Road and 100 metres north of St Johns Road"
+
+There are 57 records with this case.
+
+For this case, we need to find the intersections St. Marks and Watson Avenue, and St. Johns Road and Watson Avenue. Then find a point that is 100 metres north of the intersection of St. Johns Road and Watson Avenue.
+
+### Workflow for Special Case 1:
+1. make a point that is x metres away from intersection in the cardinal direction indicated (in other words: translate the intersection)
+2. make a line from the intersection to that point (translated point)
+3. make a buffer around the line and catch centreline segments in the buffer with the correct street name
+4. dissolve the centreline segments for each bylaw into one geom
+5. cut the dissolved segments to be from the intersection described in `btwn1` column to x metres away from that intersection in the direction specified in `direction_btwn2`.
+
+### Workflow for Special Case 2:
+1. make a point that is x metres away from intersection in the cardinal direction indicated
+2. make a line from the intersection to that point
+3. make a buffer around the line and catch centreline segments in the buffer with the correct street name
+4. dissolve the centreline segments for each community safety zone into one geom
+5.
```

Comment:
The hierarchy of information here is hard to follow, switches from numbered list to bullets. The image filenames read "case1"